### PR TITLE
Feat/wallet ticker

### PR DESCRIPTION
**Pull request by alaibe**
Add a timer of 10min which send event that client should reload data

### CODE REVIEW

File: services/wallet/reader.go [hunk 1]
```diff
@@ -89,13 +94,22 @@ func getTokenAddresses(tokens []*token.Token) []common.Address {
...
 
-func (r *Reader) Start(ctx context.Context, chainIDs []uint64) error {
-	accounts, err := r.s.accountsDB.GetAccounts()
-	if err != nil {
-		return err
-	}
-
-	return r.s.transferController.CheckRecentHistory(chainIDs, getAddresses(accounts))
+func (r *Reader) Start() error {
+	ticker := time.NewTicker(10 * time.Minute)
```
Comment (richard-ramos) [inline services/wallet/reader.go]:
```suggestion
	ticker := time.NewTicker(10 * time.Minute)
	defer ticker.Stop()
```

Comment (richard-ramos) [reply services/wallet/reader.go]: Also, probably makes sense to move the ticker inside the goroutine

File: services/wallet/reader.go [hunk 2]
```diff
@@ -89,13 +94,22 @@ func getTokenAddresses(tokens []*token.Token) []common.Address {
...
+	ctx := context.Background()
+	go func() {
+		for {
+			select {
+			case <-ctx.Done():
```
Comment (richard-ramos) [inline services/wallet/reader.go]: Is ctx.Done() called when `Logout()` is called? if not, this goroutine will run forever

File: services/wallet/reader.go (hunk 1 above)
Comment (alaibe) [reply services/wallet/reader.go]: indeed, moved to inside the coroutine

File: services/wallet/reader.go (hunk 2 above)
Comment (alaibe) [reply services/wallet/reader.go]: updated

**Review by richard-ramos — APPROVED** [review]

**Review by jrainville — APPROVED** [review]
LGTM

**Pull request merged**